### ISSUE TITLE
ardupilot installation wrong command

### ISSUE THREAD

**Issue by leoll2** [Bochicchio3/src#5]
When installing Ardupilot, the command 
`Tools/scripts/install-prereqs-ubuntu.sh -y`
doesn't work because that script is not in the `scripts` folder, but in `environment_install`.

Assuming to be in the ardupilot root dir, he correct sequence should be:
```
chmod a+x Tools/environment_install/install-prereqs-ubuntu.sh
Tools/environment_install/install-prereqs-ubuntu.sh -y
. ~/.profile
```

**Comment by emapall** [comment]
yes, apparently the installation guide changed while we were working.
http://ardupilot.org/dev/docs/building-setup-linux.html#building-setup-linux

i can remember the environment_install was not there when I first did it. 
Probably, you don't even need to make it executable, it probably already is an executable. Anyway, always refer to the official guide.
